### DISCUSSION TITLE
How can I run Nushell in Zellij?

### DISCUSSION THREAD

**Discussion by MarkuBu** [zellij-org/zellij#2855] [Q&A]
How can I run Nushell in Zellij? I use Alacritty as terminal that automatically loads Zellij. Bit I want to have Nushell inside Zellij. I only found a solution that runs Zellij inside of Nushell. Nut in this case I don't have Nushell in Zellij.

Currently I have to run Nushell in any new Zellij panel. How can I run Nushell automatically?

**Comment by Lockszmith-GH** [comment]
Did you look at [default_shell](https://zellij.dev/documentation/options.html?highlight=defaul#default_shell) ?
